### PR TITLE
mark an all-bold paragraph as a possible issue.

### PR DESCRIPTION
**Pull request by 70ray**
Do not attempt to identify section headings since the markup does not distinguish between a paragraph and a section heading, but mark an all-bold paragraph as a possible issue.

Continue to mark all-bold headings and sub-headings as definite issues.

add test

Sandbox at: https://www.pgdp.org/~rp31/c.branch/bold_para

### CODE REVIEW

**Review by srjfoo — APPROVED** [review]

File: tools/proofers/preview.js [hunk 1]
```diff
@@ -66,7 +66,6 @@ function removeTrail(txt) {
 
 const blockType = {
     PARA: 0,
-    SECHEAD: 1,
```
Comment (chrismiceli) [inline tools/proofers/preview.js:69]: since we removed this entry, should we renumber the later ones?

File: tools/proofers/preview.js [hunk 2]
```diff
@@ -544,29 +539,20 @@ window.addEventListener('DOMContentLoaded', function() {
...
-                case blockType.SECHEAD:
-                case blockType.SUBHEAD:
-                    checkHead("noBold", txt, block);
-                    break;
-                default:
-                    break;
+        function testBoldBlock(block) {
+            if((txt.indexOf("<b>", block.start) === block.start) && (txt.indexOf("</b>", block.start) === (block.end - 4))) {
+                // heading: issue, paragraph: possible issue
+                if((block.type == blockType.HEAD) || (block.type == blockType.SUBHEAD)) {
```
Comment (chrismiceli) [inline tools/proofers/preview.js:545]: *minor* no need for parentheses.

File: tools/proofers/preview.js [hunk 3]
```diff
@@ -544,29 +539,20 @@ window.addEventListener('DOMContentLoaded', function() {
...
-                switch(block.type) {
-                case blockType.HEAD:
-                case blockType.SECHEAD:
-                case blockType.SUBHEAD:
-                    checkHead("noBold", txt, block);
-                    break;
-                default:
-                    break;
+        function testBoldBlock(block) {
+            if((txt.indexOf("<b>", block.start) === block.start) && (txt.indexOf("</b>", block.start) === (block.end - 4))) {
```
Comment (chrismiceli) [inline tools/proofers/preview.js:543]: the indices we see (`- 4`, `+ 3`, `+ 2`) could be easier if we explained them some, or computed them from a fixed value? Like:
```
if((txt.indexOf("<b>", block.start) === block.start) &&
   (txt.indexOf("</b>", block.start) === (block.end - "</b>".length))) {
```
obviously subjective, just thinking out loud.

**Review by chrismiceli — APPROVED** [review]

File: tools/proofers/preview.js (hunk 1 above)
Comment (70ray) [reply tools/proofers/preview.js:69]: I don't think we need to since they are just arbitrary numbers.

File: tools/proofers/preview.js (hunk 3 above)
Comment (70ray) [reply tools/proofers/preview.js:543]: That seems like a good idea.

File: tools/proofers/preview.js (hunk 1 above)
Comment (chrismiceli) [reply tools/proofers/preview.js:69]: No, definitely not required, so makes sense to leave as it to reduce risk of change.

**Pull request merged**